mingw-w64-qt6-static/PKGBUILD

```
@@ -138,12 +138,18 @@ build() {
   [[ -d B${MSYSTEM} ]] && rm -rf B${MSYSTEM}
   mkdir -p B${MSYSTEM} && cd B${MSYSTEM}
 
+  declare -a _extra_config
+  if [[ ${CARCH} != aarch64 ]]; then
+    _extra_config+=('-G"Ninja Multi-Config"' '-DCMAKE_CONFIGURATION_TYPES="Release;Debug"')
```

```suggestion
    _extra_config+=('-GNinja Multi-Config' '-DCMAKE_CONFIGURATION_TYPES=Release;Debug')
```

---

I don't know about what cmake wants it to be, might need to be
```suggestion
    _extra_config+=('-G' 'Ninja Multi-Config' '-DCMAKE_CONFIGURATION_TYPES=Release;Debug')
```
To match what it used to be before the array